src/App.js

```diff
@@ -1,37 +1,66 @@
 
 import './App.css';
-import Nav from './Nav';
+import Nav from './components/Nav';
+import Form from './components/Form';
+import { useState } from 'react';
+
+// calling the Stripe API outside of the function and outside of useEffect because can't get await/using a promise to work inside of those (not something we covered in class)
+const stripe = require('stripe')('sk_test_51HTHi2Hvy5NeCPEsFMyNF55MzJSDciOw2Qmp4KS4DLaf51eFUxDQcDu4dHfOc7PcqmyTbFHfKh3GhjhVDfmmWE21009ZNmVRCE');
+
+const payouts = await stripe.payouts.list({
+  limit: 1000
+});
 
 function App() {
 
+  //Credit: Tried to adapt Esther's recent Unsplash API lesson to make the below work.
+
+  //Creating a stateful variable to hold the user's date choice in the form
+  const [chosenDate, setChosenDate] = useState(); 
+
+  //this function is meant to handle form submission & filtering the date choice and its related amount
+  const getDate = (event, chosenDate) => {
+    event.preventDefault();
+
+    //trying to loop over the date array to return only the data for the date selected in the form
+    
+    const filteredDate = chosenDate.filter
+    
+    // (correctSelection => {
+    //   return date === chosenDate
+    // })
+
+    setChosenDate(filteredDate);
+
+  }
+
   return (
-    <div>
+    <div className="App">
       <Nav />
+      <Form setChosenDate />
       <header>
         <h2>Date Received</h2>
-        <h2>Net Gift Amount</h2>
+        <h2>Net Gift Amount (in USD)</h2>
       </header>
+
       {payouts.data.map((data) => {
 
         let date = new Date(data.arrival_date * 1000)
-
         let amount = data.amount / 100;
 
         return (
+          //here I'm returning all of the date and amount data available to display it in columns, because I realized this would be more useful for our business needs
           <ul className="columns">
```

Comment:
For <ul> element there is a warning in the console, which you can get rid of by adding a unique key prop such as:
<ul key={data.id} className="columns">

"Warning: Each child in a list should have a unique "key" prop."

Reply:
It would also make sense to move this part of the app into a different component in the future as the app progresses.